packages/cli/src/create/utils.ts

```diff
@@ -0,0 +1,344 @@
+  const configPathTS = path.join(cwd, "apibara.config.ts");
+
+  if (fs.existsSync(configPathJS)) {
+    return "javascript";
+  }
+  if (fs.existsSync(configPathTS)) {
+    return "typescript";
+  }
+
+  throw new Error(red("✖") + " No apibara.config found");
+}
+
+export function getDnaUrl(chain: Chain, network: Network) {
+  if (chain === "ethereum") {
+    if (network === "mainnet") {
+      return dnaUrls.ethereum;
+    }
+    if (network === "sepolia") {
+      return dnaUrls.ethereumSepolia;
+    }
+  }
+
+  if (chain === "beaconchain") {
+    if (network === "mainnet") {
+      return dnaUrls.beaconchain;
+    }
+  }
+
+  if (chain === "starknet") {
+    if (network === "mainnet") {
+      return dnaUrls.starknet;
+    }
+    if (network === "sepolia") {
+      return dnaUrls.starknetSepolia;
+    }
+  }
+
+  throw new Error(red("✖") + " Invalid chain or network");
+}
+
+/**
+ * Converts a kebab-case string to camelCase.
+ *
+ * Examples:
+ * - "hello-world" → "helloWorld"
+ * - "my-long-variable-name" → "myLongVariableName"
+ * - "MY-CAPS" → "myCaps"
+ * - "-leading-dash" → "leadingDash"
+ * - "trailing-dash-" → "trailingDash"
+ * - "double--dash" → "doubleDash"
+ * - "hello---world" → "helloWorld"
+ * - "mixed_dash-and_underscore" → "mixedDashAndUnderscore"
+ *
+ * @param str The kebab-case string to convert
+ * @returns The camelCase version of the string
+ */
+export function convertKebabToCamelCase(_str: string): string {
+  let str = _str;
+
+  // Handle empty or invalid input
+  if (!str || typeof str !== "string") {
+    return "";
+  }
+
+  // Check if already camelCase
+  if (/^[a-z][a-zA-Z0-9]*$/.test(str)) {
+    return str;
+  }
+
+  // Trim leading/trailing dashes and spaces
+  str = str.trim().replace(/^-+|-+$/g, "");
+
+  // Handle empty string after trim
+  if (!str) {
+    return "";
+  }
+
+  return (
+    str
+      // Replace multiple consecutive dashes/underscores with a single dash
+      .replace(/[-_]+/g, "-")
+      // Split on dash
+      .split("-")
+      // Filter out empty strings (from consecutive dashes)
+      .filter(Boolean)
+      // Convert each word
+      .map((word, index) => {
+        // Convert word to lowercase
+        const _word = word.toLowerCase();
+
+        // Capitalize first letter if not the first word
+        if (index > 0) {
+          return _word.charAt(0).toUpperCase() + _word.slice(1);
+        }
+
+        return _word;
+      })
+      .join("")
+  );
+}
+
+export async function checkFileExists(
+  path: string,
+  askPrompt = false,
+  fileName?: string,
+): Promise<{
+  exists: boolean;
+  overwrite: boolean;
+}> {
+  if (!fs.existsSync(path)) {
+    return {
+      exists: false,
+      overwrite: false,
+    };
+  }
+
+  if (askPrompt) {
+    const { overwrite } = await prompts({
+      type: "select",
+      name: "overwrite",
+      message: `${fileName ?? basename(path)} already exists. Please choose how to proceed:`,
+      initial: 0,
+      choices: [
```

Comment:
There should be an option (in some cases only?) to ignore creating the file but to continue the process.

My use case is adding a second indexer to the same project and specifying drizzle again, I don't want to overwiret the file but I also don't want the process to stop!

![image](https://github.com/user-attachments/assets/37183cdd-0b05-4ace-8b58-2d6d93feeb2d)